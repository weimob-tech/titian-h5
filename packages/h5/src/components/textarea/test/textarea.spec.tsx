import { newSpecPage } from '@stencil/core/testing';
import { h } from '@stencil/core';
import { TiTextarea } from '../index';
import { TestContainer } from '../../common/test/container';

describe('渲染 ti-steps', () => {
  it('基本用法', async () => {
    const page = await newSpecPage({
      components: [TiTextarea, TestContainer],
      template: () => (
        <test-conatiner>
          <ti-textarea placeholder="请输入评论文字，限200字以内…" />
          <ti-textarea auto-height />
          <ti-textarea show-count />
        </test-conatiner>
      ),
    });
    expect(page.root).toMatchSnapshot();
  });
});
