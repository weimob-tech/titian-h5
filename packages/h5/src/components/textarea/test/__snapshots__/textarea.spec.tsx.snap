// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`渲染 ti-steps 基本用法 1`] = `
<ti-textarea>
  <mock:shadow-root>
    <div class="titian-textarea" part=""><textarea class="titian-textarea-box" maxlength="140" part="" placeholder="请输入评论文字，限200字以内…"></textarea>
    </div>
  </mock:shadow-root>
</ti-textarea>
`;
